### PR TITLE
Add serializer achievements

### PR DESCRIPTION
**Pull request by joelwurtz**

### CODE REVIEW

File: achievements/serializer.md [hunk 1]
```diff
@@ -1 +1,56 @@
+# Symfony Serializer Refactoring Plan
 
+The symfony/serializer component has several normalizer implementations which are rather redundant and hard to maintain. The idea is to refactor the whole process of translating between array and objects and back and rely on PropertyAccess for the logic of how to interact with objects. The discussion is summarized at https://github.com/symfony/symfony/issues/30818.
+
+EUFOSSA Hackathon helps us a lot to validate this discussion, and began the refactoring of the serializer, here is a detailed list on what we have achieved in regards to the plan during this two days:
```
Comment (javiereguiluz) [inline achievements/serializer.md]: `helps us a lot` -> `helped us a lot` ?

Comment (javiereguiluz) [inline achievements/serializer.md]: `this two days` -> `these two days`

File: achievements/serializer.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+
+## Tests - In review
+
+We refactored the tests so that we can ensure a consistent behaviour and test the new normalizer when it is implemented. In that process, we found a couple of bugs that we fixed in Symfony 3.4 where applicable, and 4.2 for newer features.
+
+See https://github.com/symfony/symfony/pull/30888
+
+## Properties extraction - Merged
+
+A new extension point was provided to be able to extract properties from an object (and not only for a class). This is needed in order to implement future behaviour that need the data of the object to filter the property list.
```
Comment (javiereguiluz) [inline achievements/serializer.md]: `that need` -> `that needs`

**Review by javiereguiluz — COMMENTED** [review]
Nice work! Congrats and thank you all.

**Review by jfhovinne — APPROVED** [review]
Nice work, thanks!
Will fix some typos.

**Review by jewome62 — COMMENTED** [review]
:+1:

**Pull request merged**